lib/server.js

```diff
@@ -64,7 +64,14 @@ function Server (opts) {
   // initialize websocket server
   if (~this.transports.indexOf('websocket')) {
     // keep require('ws') as separate expression for packers (browserify, etc)
-    var WebSocketServer = (this.wsEngine ? require(this.wsEngine) : require('ws')).Server;
+    var wsModule;
+    try {
+      wsModule = require(this.wsEngine);
+    } catch (ex) {
+      this.wsEngine = 'ws';
+      wsModule = require(this.wsEngine);
```

Comment:
According to the comment above `// keep require('ws') as separate expression for packers (browserify, etc)`, shouldn't it rather be `wsModule = require('ws');` here?

Related: https://github.com/socketio/engine.io/pull/418